Formula/sshs.rb

```diff
@@ -0,0 +1,18 @@
+class Sshs < Formula
+  desc "Graphical command-line client for SSH"
+  homepage "https://github.com/quantumsheep/sshs"
+  url "https://github.com/quantumsheep/sshs/archive/refs/tags/1.3.1.tar.gz"
+  sha256 "cc6cdfe6be6f3efea29c8acc5a60ea037a88e445bced1619e7002a851d9bbe88"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
```

Comment:
```suggestion
```
You don't actually use `gmake`, so this isn't needed.

Reply:
I'm using `make`, doesn't this need the `make` formula ?

Reply:
No, `make` refers to the one provided by the system. The formula's executable is called `gmake`.
```
❯ brew info make
make: stable 4.3 (bottled)
Utility for directing compilation
[snip]
==> Caveats
GNU "make" has been installed as "gmake".
If you need to use it as "make", you can add a "gnubin" directory
to your PATH from your bashrc like:

    PATH="/usr/local/opt/make/libexec/gnubin:$PATH"
[snip]
```